libraries/botbuilder-lg/src/templates.ts

```diff
@@ -130,6 +137,7 @@ export class Templates implements Iterable<Template> {
         this.importResolver = importResolverDelegate;
         this.options = options || [];
         this.source = source;
+        this.namedReferences = namedReferences || {};
```

Comment:
`this.namedReferences = nameReferences ?? {};`

Reply:
Optional chain is not support in ts 3.5.

Reply:
Ah - good point, didn't realize we were back a version or two in this package.